### PR TITLE
Add grplot.yml

### PR DESCRIPTION
**Pull request by ghiffaryr**

### CODE REVIEW

File: packages/grplot.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+repo: 'ghiffaryr/grplot'
+section: 'domain specific libraries'
+description: 'lazy statistical data visualization.'
```
Comment (story645) [inline packages/grplot.yml:3]:
```suggestion
description: 'Lazy statistical data visualization.'
```
Everything else is caps, so would be more standard- also is there anything more descriptive than lazy?

**Review by story645 — CHANGES_REQUESTED** [review]
Thanks for adding the listing!

File: packages/grplot.yml (hunk 1 above)
Comment (ghiffaryr) [reply packages/grplot.yml:3]: Okay, thanks, and nope since it is intended for lazy people to draw complete and attractive statistical graphs

**Review by timhoffm — COMMENTED** [review]
Semi-off-topic: I recommend that you expand your project documentation. As is, it's very hard for a user to find out what he can do via your interface and what parameters to use. They essentially need to read through the whole list of parameters. One improvement would be e.g. to group the parameters into sections by topic.

File: packages/grplot.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+repo: 'ghiffaryr/grplot'
+section: 'domain specific libraries'
+description: 'Lazy statistical data visualization.'
```
[the remark this answers is not in the document]
Comment (ghiffaryr) [reply packages/grplot.yml:3]: > Semi-off-topic: I recommend that you expand your project documentation. As is, it's very hard for a user to find out what he can do via your interface and what parameters to use. They essentially need to read through the whole list of parameters. One improvement would be e.g. to group the parameters into sections by topic.

Sure, It's already on my plan. It takes time for me to finish all by myself. So, I will make it gradually.

[the remark this answers is not in the document]
Comment (ghiffaryr) [reply packages/grplot.yml:3]: > 

I'm thinking about a new description. What about this? Complete and attractive statistical graphs in one function.

[the remark this answers is not in the document]
Comment (timhoffm) [reply packages/grplot.yml:3]: Sounds good.

**Review by timhoffm — COMMENTED** [review]
The description is good now. I'm still a bit hesitant to merge as the library is still in quite an early stage and I have the impression it's not yet documented enough to be usable. @ghiffaryr What is your time plane for future work on the library?

**Comment by story645** [comment]
I (as twitter:matplotlib) suggested that @ghiffaryr submit the PR & sorry to you both if the library wasn't quite ready for that.

Maybe we should separately sort out the baseline criteria for submissions to this repo?

**Comment by ghiffaryr** [comment]
> The description is good now. I'm still a bit hesitant to merge as the library is still in quite an early stage and I have the impression it's not yet documented enough to be usable. @ghiffaryr What is your time plane for future work on the library?

Anyway, thank you so much for the advice. This weekend, I will continue to work on it because I have my own job too.

**Comment by ghiffaryr** [comment]
@timhoffm Full Documentation and some updates were just done

**Review by timhoffm — APPROVED** [review]

**Pull request merged**